docs/getting-started.md

```diff
@@ -7,22 +7,29 @@ description: Jump in and get started quickly using Momento Serverless Cache with
 ---
 
 # Getting Started with Momento Serverless Cache
-If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. There are multiple ways to access the service below. 
+If you're looking to jump in and get started with Momento Serverless Cache with an SDK or CLI, you've come to the right place. Want to quickly get a feel for the Momento experience without installing any software? You can do so right here [in your browser](#try-our-cli-and-an-sdk-in-your-browser). Or, if you're ready to setup your development environment and get to building your own applications with Momento you can follow the [simple instructions below](#install-the-momento-command-line-interface-cli).
 
-## Try our CLI in your browser
+## Try our CLI and an SDK in your browser
 
-You can try out our CLI tool today, right in your browser.  No software installation required!
+You can try out Momento Serverless Cache today, right in your browser. No software installation required! **Note: Momento Serverless Cache is an API based serverless service. It does not deploy anything into your (AWS, GCP, Azure, etc.) account.**
 
-Head over to our [Try Momento Serverless Cache page](https://www.gomomento.com/try-momento-for-free), and
-you can experiment with Momento CLI commands in a shell session directly inside your browser.
-You can create a free auth token, create a cache, and start running `set` and `get` commands
-on your cache. It only takes a few minutes!
+First, request your free auth token, create a cache, configure your CLI, and start running `set` and `get` commands on your cache. Launch below.
 
-**Note: Momento Serverless Cache is an API based serverless service. It does not deploy anything into your (AWS, GCP, Azure, etc.) account.**
+<a href="https://play.instruqt.com/embed/momento/tracks/sandbox-container-1challenge?token=em_14J4EucaTHzL0rXw&finish_btn_target=_top&finish_btn_text=Return+to+Docs&finish_btn_url=https%3A%2F%2Fdocs.momentohq.com%2Fgetting-started#try-our-cli-and-an-sdk-in-your-browser" target="_top"><img src="/img/cli_lab.png" alt="CLI lab" /></a>
```

Comment:
@pete-naylor thanks for this!  I think this will work better than the embed.

Does this open in a new tab or just load over the current docs tab in your browser?  If the latter, curious if you have any thoughts about doing it in a new tab vs. current tab?

cc:  @eaddingtonwhite 

thanks again for working to get these in!

Reply:
I fiddled around with some variations on this last night and this seemed to be the best approach. Gets a nice full screen view, and returns to the right place with the custom button you suggested. It opens in the same tab at _top - I like this as it keeps everything in the same flow without introducing tab forks in the road. I was concerned that somehow people may not make it back to the original getting started page if it's a separate tab. Seems like there's a good chance that this will get reworked by Hannah and others who might have better ideas - I just needed it to be functional for the moment and I am by no means an expert on this stuff!!